src/cleanfigure.m

```diff
@@ -96,7 +96,7 @@ function cleanfigure(varargin)
 
```

Comment:
BTW how come, that we have this check against quiverplots? It seems fairly ineffective.